File: levels/level5/task.cpp
// Aufgabe: Schreibe ein Programm, das zwei Zahlen multipliziert und das Ergebnis ausgibt.
// Dein Code beginnt hier.

#include <iostream>

int main() {
    int zahl1, zahl2, produkt;
    std::cout << "Gib die erste Zahl ein: ";
    std::cin >> zahl1;
    std::cout << "Gib die zweite Zahl ein: ";
    std::cin >> zahl2;
    produkt = zahl1 * zahl2;
    std::cout << "Das Produkt ist: " << produkt << std::endl;
    return 0;
}

File: levels/level16/task.py
# Aufgabe: Erstelle ein Struktogramm für ein Programm, das mehrere Funktionen enthält (z.B. eine Funktion zur Berechnung der Summe und eine zur Berechnung des Produkts von zwei Zahlen).
# Dein Code beginnt hier

def summe_berechnen(zahl1, zahl2):
    return zahl1 + zahl2

def produkt_berechnen(zahl1, zahl2):
    return zahl1 * zahl2

def main():
    # Eingabe der ersten Zahl
    zahl1 = int(input("Geben Sie die erste Zahl ein: "))
    # Eingabe der zweiten Zahl
    zahl2 = int(input("Geben Sie die zweite Zahl ein: "))

    # Berechnung und Ausgabe der Summe
    summe = summe_berechnen(zahl1, zahl2)
    print(f"Die Summe ist: {summe}")

    # Berechnung und Ausgabe des Produkts
    produkt = produkt_berechnen(zahl1, zahl2)
    print(f"Das Produkt ist: {produkt}")

if __name__ == "__main__":
    main()
